tiatoolbox/annotation/storage.py

```diff
@@ -2996,6 +3099,7 @@ def remove_many(self, keys: Iterable[str]) -> None:
             self.con.commit()
 
     def __setitem__(self, key: str, annotation: Annotation) -> None:
+        """Implements a method to assign a value to and item."""
```

Comment:
typo, should be: to an item

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198